index.ipynb

```diff
@@ -94,48 +105,48 @@
    "cell_type": "markdown",
    "metadata": {},
    "source": [
-    "## Contents\r\n",
```

Comment:
Hm. It seems my machine creates Unix style line endings whilst previous version used Windows style. I would have thought nbdev's notebook cleaning tools should have normalized this? Now just about everything has "changed".